mozci/push.py

```diff
@@ -155,7 +155,17 @@ def tasks(self):
         )
 
         def add(data):
-            for task in data:
+            if "header" in data:
+                data["data"] = [
+                    {
+                        field: entry[i]
+                        for i, field in enumerate(data["header"])
+                        if entry[i] is not None
+                    }
+                    for entry in data["data"]
```

Comment:
use `zip` instead if an index (`i`):

```
[
    {
        field: value
        for field, value in zip(data['header'], entry) 
        if value is not None
    } 
    for entry in data['data'] 
]
```

or, if you do not care about extra Nones:

```
[dict(zip(data['header'], entry)) for entry in data['data']]
```

Reply:
Done with 018f2b3.